soh/soh/Enhancements/randomizer/location_access/overworld/lon_lon_ranch.cpp

```diff
@@ -11,7 +11,7 @@ void RegionTable_Init_LonLonRanch() {
     }, {
         //Locations
         LOCATION(RC_SONG_FROM_MALON,     logic->IsChild && logic->HasItem(RG_ZELDAS_LETTER) && logic->HasItem(RG_FAIRY_OCARINA) && logic->AtDay),
-        LOCATION(RC_LLR_GS_TREE,         logic->IsChild),
+        LOCATION(RC_LLR_GS_TREE,         logic->IsChild && logic->CanBonkTrees() && logic-> CanAttack()),
```

Comment:
```suggestion
        LOCATION(RC_LLR_GS_TREE,         logic->IsChild && logic->CanBonkTrees()),
```
Some GS_TREE checks lack `CanAttack()` since you can kill with a pot in those areas

Reply:
Ah, that's right, I'll get on it!